src/templates/bespoke/presenter/presenter-view.tsx

```diff
@@ -73,9 +93,26 @@ const presenterView = (deck) => {
     return $cache[klass]
   }
 
+  const resizeNotes = (direction: string) => {
+    const current = parseFloat($(classes.noteContainer).style.fontSize) || 1
+    const intended = current + (direction === 'bigger' ? 0.1 : -0.1)
+    $(classes.noteContainer).style.fontSize = `${intended.toFixed(1)}em`
+  }
```

Comment:
Using CSS custom property would be smart as like as https://github.com/marp-team/marp-cli/pull/427#discussion_r810452936.

```typescript
const notesFontSizeBase = 0
const resizeNotesFont = (increment: number) => {
  notesFontSizeBase = Math.max(-0.5, increment + notesFontSizeBase) // Prevent making too smaller
  $(classes.noteContainer).style.setProperty('--bespoke-marp-note-font-base', notesFontSizeBase.toFixed(1))
}

$(classes.noteButtonsBigger).addEventListener('click', () => resizeNotesFont(0.1))
$(classes.noteButtonsSmaller).addEventListener('click', () => resizeNotesFont(-0.1))
```

```css
.bespoke-marp-note {
  /* Base font size is 1.1em, and add modified difference of font size by user */
  font-size: calc((1.1 + var(--bespoke-marp-note-font-base, 0)) * 1em);
}
```